next.config.js

```diff
@@ -2,6 +2,8 @@
 const nextConfig = {
   reactStrictMode: true,
   images: {
+    domains: ['d2szwvl7yo497w.cloudfront.net'], // Add your domain here
```

Comment:
can u remove this? I guess this was only for testing

Reply:
done